container/README.md

```diff
@@ -13,20 +13,141 @@ If a section or action is tagged as "Fedora", it should be applied only for a
+ln -s os-autoinst-distri-opensuse opensuse
+git clone https://github.com/os-autoinst/os-autoinst-needles-opensuse.git
+mv os-autoinst-needles-opensuse/ opensuse/products/opensuse/needles
+```
+
+NOTE: To execute opensuse test the worker needs some dependencies, install them as follow:
+    
+    docker-compose exec worker zypper in -y os-autoinst-distri-opensuse-deps
+
+To launch a job using clone_job, execute: 
+
+    docker-compose webui /usr/share/openqa/script/clone_job.pl --apikey 1234567890ABCDEF --apisecret 1234567890ABCDEF --host nginx https://openqa.opensuse.org/tests/1613965
+
+
+### Change the number web UI replicas (optional)
+
+To set the number of replicas set the environment variable
+`OPENQA_WEBUI_REPLICAS` to the desired number. If this is not set, then the
+default value is 2.
+
+    export OPENQA_WEBUI_REPLICAS=3
+
+Or, you can set the variable in-line 
+    
+    OPENQA_WEBUI_REPLICAS=3 docker-compose up -d
+
+Additionally you can edit the .env file to set the default value for this
+variable.
+
+### Change the container images (optional)
+
+By default the the docker-compose will use the images of openqa_webui, 
+openqa_webui_lb and openqa_worker stored in registry.opensuse.org.
+
+The selection of other images can be done setting up the environment variables: 
+OPENQA_WEBUI_IMAGE, OPENQA_WEBUI_LB_IMAGE, and OPENQA_WORKER_IMAGE
+
+For instance, to build and use local images follow these steps
+
+    $ docker-compose build
+    $ OPENQA_WEBUI_IMAGE=openqa_webui OPENQA_WEBUI_LB_IMAGE=openqa_webui_lb OPENQA_WORKER_IMAGE=openqa_worker docker-compose up -d
+
+### Add the exported port for the load balancer (optional)
+
+By default the load balancer exposes the web UI on ports 9526, 80 and 443.
+
+```
+ports:
+  - "9526:9526"
+  - "80:9526"
+```
+
+### Enable the SSL access to the load balancer (optional)
+
+Enable the SSL access in three steps:
+
+1. To expose the SSL port uncomment this line in the docker-compose.yaml file
+   in the service nginx.
+
+```
+- "443:443"
+```
+
+You can change the exported port if 443 is already used in your computer, for
+instance:
+
+```
+- "10443:443"
+```
+
+2. Provide an SSL certificate.
+
+```
+- cert.crt:/etc/ssl/certs/openqa.crt
+- cert.key:/etc/ssl/certs/openqa.key
+```
+
+3. Modify nginx.cfg to use this certificate. Uncomment the lines
+
+```
+ssl_certificate     /etc/ssl/certs/openqa.crt;
+ssl_certificate_key /etc/ssl/certs/openqa.key;
+```
+
+### Use OpenID auth method (optional)
+
+By default the auth method is Fake. But OpenID can be selected modifying the variable method in the auth section of conf/openqa.ini file.
+
+That implies that by default the api/key won't exists. So the setup has to be done in two steps. 
+
+1. First run the docker-compose up
+2. Then enter to the web UI page and do Login
+3. A new api/key is created. Check it out in the db container
+
+        $ docker-compose exec db psql -U openqa openqa
+        openqa=# SELECT * FROM api_keys;
```

Comment:
I like to have documentation for accessing the database within the container. However, I'm wondering whether this is really necessary for simply viewing the API credentials at this point. Wouldn't it be possible to simply show them within the web UI?

Reply:
Yes, you're right, better the UI